src/components/navbar/Navbar.jsx

```diff
@@ -19,20 +19,31 @@ function Navbar (){
         </div>
         <div className="navbar-links flex-center" >
 
-            {authState.loginStatus? <span className="nav-login-link"> <button className="nav-login-btn" onClick={()=> authDispatch({ type:'LOGOUT' })} >Log out</button> </span> :<Link className="nav-login-link" to="/login"><button className="nav-login-btn">Login</button></Link>}
+            {authState.loginStatus? 
+                    <span className="nav-login-link"> 
+                        <button className="nav-login-btn" onClick={()=> authDispatch({ type:'LOGOUT' })} >Log out</button>
+                    </span> :
+                    <Link className="nav-login-link" to="/login">
+                        <button className="nav-login-btn">Login</button>
+                    </Link>
+            }
                         
             <ul className="nav-list">
                 <li className="nav-list-item">
-                    <Link className="nav-list-link icon-badge pos-rel" to="./screens/html/wishlist.html"><i
+                    <div className="nav-list-link icon-badge pos-rel"  style={{ cursor:"pointer" }} ><i
```

Comment:
Avoid inline-style instead can make a class to reuse

Reply:
thanks for pointing out i will correct it